website/docs/docs/dbt-cloud-apis/sl-python-sdk.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Python SDK"
+id: sl-python
+description: "Learn how to use the dbt Semantic Layer SDK for Python library to interact with the dbt Semantic Layer."
```

Comment:
We're calling it the Python SDK everywhere else, but the Semantic Layer SDK here. Might be a little confusing